README.md

```diff
@@ -54,13 +54,57 @@ Windows.
    * Add `/t:rebuild` for a clean rebuild instead of an incremental build.
 
 If successful, the driver binary (`IntelHaxm.sys`) will be generated in
-`X:\path\to\haxm\obj\out\win7\x64\` (or `X:\path\to\haxm\obj\out\win7\x86\` if
+`X:\path\to\haxm\obj\out\Win7\x64\` (or `X:\path\to\haxm\obj\out\Win7\x86\` if
 `Platform="Win32"`), and will be able to run on Windows 7 and later.
 
+## Testing on Windows
+### System Requirements
+Note that these are requirements for the _test_ environment, which does not
+have to be the same as the _build_ environment.
+
+1. An Intel CPU that supports Intel VT-x with _Extended Page Tables_ (EPT).
+   * [Here][intel-ept-cpus] is a list of CPUs that meet this requirement. As a
+rule of thumb, if you have an Intel Core i3, i5, i7 or i9 (any generation), you
+are good to go.
+   * EPT is an advanced feature of Intel VT-x. CPUs that support EPT also
+support _Unrestricted Guest_ (UG), which is another advanced feature of VT-x.
+It may still be possible to run HAXM on very old (pre-2010) CPUs, e.g.
+Intel Core 2 Duo, which implement an earlier version of VT-x that does not
+include either EPT or UG, because there is still legacy code that enables HAXM
+to work in non-EPT and non-UG modes. However, such code may be removed soon.
+1. Windows 7 or later; both 32-bit and 64-bit Windows are supported.
+   * Running HAXM in a nested virtualization setup, where Windows itself runs as
+a guest OS on another hypervisor, may be possible, but this use case is not well
+tested.
+
+### One-time setup
+The following steps prepare the test environment for installing a test-signed
+`IntelHaxm.sys`, i.e. one that is built using the `Debug` configuration. For
+more details, please read [this article][windows-test-driver-install].
+
+1. Disable Hyper-V and enable _Test Mode_:
+   1. Open an *elevated* Command Prompt.
+   1. `bcdedit /set hypervisorlaunchtype off`
+   1. `bcdedit /set testsigning off`
+   1. Reboot.
+1. Install the test certificate:
+   1. Copy `IntelHaxm.cer` from the build environment to the test environment
+(if the two are not the same). This file is generated alongside `IntelHaxm.sys`
+by the `Debug` build configuration.
+   1. In the test environment, open an *elevated* Command Prompt and run
+`certmgr.exe /add X:\path\to\IntelHaxm.cer /s /r localMachine root`
+
+### Loading and unloading the test driver
+TODO: HaxmLoader.exe usage
```

Comment:
I think we need to add how to load haxm driver here, or else they could build haxm driver and couldn't use it?

Reply:
See the commit log. The plan is to release that tool first, so I can provide a download link here.